Model/Refund/Push.php

```diff
@@ -119,7 +119,7 @@ public function __construct(
     }
 
     /**
-     * This is called when a refund is made in Buckaroo Payment Plaza.
+     * This is called when a refund is made in Buckaroo Buckaroo Plaza.
```

Comment:
maybe change this one to Buckaroo **Payment** Plaza

Reply:
**This should be Buckaroo Plaza:**
 * This is called when a refund is made in the Buckaroo Plaza.